frontend/pages/linkout.vue

```diff
@@ -7,7 +7,7 @@
                 <v-btn @click="$router.back()">
                     {{ $t('linkout.abort') }}
                 </v-btn>
-                <v-btn :href="target">
+                <v-btn :href="target" class="linkoutContinueButton">
```

Comment:
this should probably just be an `outline` button instead of needing that custom css class.